_data/cloud.yml

```
@@ -91,7 +91,9 @@ websites:
     img: heroku.png
     tfa:
       - totp
-    doc: https://devcenter.heroku.com/articles/two-factor-authentication
+      - proprietary
+      - hardware
```

Shouldn't this be `u2f` instead?
```suggestion
      - u2f
```

---

> You will need a Security Key based on WebAuthn standard.

https://devcenter.heroku.com/articles/security-keys